message_definitions/v1.0/common.xml

```diff
@@ -377,7 +377,7 @@
                <entry value="153" name="MAV_COMP_ID_SERVO14">
                     <description/>
                </entry>
-               <entry value="154" name="MAV_COMP_ID_GIMBAL">
+               <entry value="154" name="MAV_COMP_ID_SOLO_GIMBAL">
```

Comment:
I believe this ID was not created for the Solo Gimbal, but for generic Mavlink aware Gimbal controllers like the STorM32